packages/design-system-twrnc-preset/src/Theme/Theme.utilities.ts

```diff
@@ -41,7 +71,19 @@ export const generateTailwindConfig = (
         ...baseConfig.theme?.extend,
         colors: {
           ...baseConfig.theme?.extend?.colors,
-          ...flattenedColors,
+          ...themeColors,
+        },
+        fontSize: {
+          ...typographyTailwindConfig.fontSize,
+        },
+        fontFamily: {
+          ...typographyTailwindConfig.fontFamily,
+        },
+        letterSpacing: {
+          ...typographyTailwindConfig.letterSpacing,
+        },
+        lineHeight: {
+          ...typographyTailwindConfig.lineHeight,
```

Comment:
@georgewrmarshall let me know if you prefer it this way or just `...typographyTailwindConfig`. This way prevents ANYTHING to be added to `typographyTailwindConfig` which may affect the final config if not done correctly